tasks/enable.yml

```diff
@@ -0,0 +1,104 @@
+---
+- name: Check trust compatibility
+  fail:
+    msg: Fapolicyd does not support trust setting fapolicyd_setup_trust
```

Comment:
```suggestion
    msg: >-
      Fapolicyd does not support trust setting fapolicyd_setup_trust on EL
      version < 8.3
```

Reply:
And please do the same with below checks, it'd be nice to say on what version exactly this is not supported